### PR TITLE
Add background documentation to the Readme.

### PR DESCRIPTION
**Pull request by ithake**
No new blog post, I'm afraid!  Just some learns from installing locally on windows.

I've successfully installed using chocolatey in place of apt-get - hence the added background - but wasn't confident enough in my process to add it here. I passed on my instructions to a colleague so if it worked for them I might add them in a future PR.

### Contents ###
First commit:
- .gitignore - exclude intellij .iml files.
- fixed broken link to confluence.

Second commit:
- adding some background to a new "Technical Stack" section and installation instructions.

---

~Please add a direct link to your post here:~

~https://<your-username>.github.io/blog/<post-url>~

Have you (please tick each box to show completion):

 - ( N/A ) Added your blog post to a single category?
 - ( NA )Added a brief summary for your post? Summaries should be roughly two sentences in length and give potential readers a good idea of the contents of your post.
 - [X] Checked that the build passes?
 - [X] Checked your spelling (you can use `npm spellcheck` if that's your thing)
 - ( N/A ) Ensured that your author profile contains a profile image, and a brief description of yourself? (make it more interesting than just your job title!)
 - [X] Optimised any images in your post? They should be less than 100KBytes as a general guide.

Posts are approved based on their category. replace `(at)` with `@` to notify them!

 - Tech - Colin Eberhardt - @ColinEberhardt 
 - Cloud - Colin Eberhardt - (at)ColinEberhardt
 - Delivery - Matt Phillips - (at)scottlogicmphillips
 - UX - Graham Odds - (at)godds
 - People - Graham Odds - (at)godds
 - Testing - Laurence Pisani - (at)lpisani-SL

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +39,22 @@ This gets the repository down to ~8MB and ~150 files (whereas checking out all a
...
+
+The blog consists of static HTML pages with content generated using: 
+- [github-pages][ruby-github-pages] for deployment hooks
+- [Jekyll][jekyll-docs] for static site generation generator
+- [Kramdown][kramdown-syntax] for an extended markdown syntax
+- [Liquid][ruby-liquid] for templating functionality
+- [Nokogiri][ruby-nokogiri] for efficient XML and HTML handling, relying on:
+  - Native XML Parsers
+- [Bundler][ruby-bundler] to manage gems and dependencies
+- [Ruby][ruby-downloads].
```
Comment (ithake) [inline README.md:56]: "and in the darkness bind them."

File: README.md [hunk 2]
```diff
@@ -78,3 +106,20 @@ To minify SCSS, run:
 npm run style
 ```
 
+[confluence-getting-started]: https://scottlogic.atlassian.net/wiki/spaces/INT/pages/3577479175/Getting+started+with+the+Scott+Logic+blog
```
Comment (ithake) [inline README.md:113]: FYI - this is the new link - previous link: https://scottlogic.atlassian.net/wiki/spaces/INT/pages/219054172/Getting+started+with+the+blog was broken.

File: README.md [hunk 3]
```diff
@@ -1,12 +1,23 @@
-## Scott Logic Blogs
+# Scott Logic Blogs
 
-See the [confluence page for user instructions](https://scottlogic.atlassian.net/wiki/spaces/INT/pages/219054172/Blog+Publishing), and use the [blog tool](https://cz90l8ad7e.execute-api.eu-west-2.amazonaws.com/production/) for an easy way to publish blog posts 
+See the [confluence page for user instructions][confluence-getting-started], 
+and use the [blog tool][blog-tool-endpoint] for an easy way to publish blog posts.
```
Comment (ColinEberhardt) [inline README.md]: yikes - the 'blog tool' hasn't been supported for a while now, and isn't mentioned in our confluence pages. Can you please remove references to it, and I'll look at decommissioning it properly.

**Review by ColinEberhardt — CHANGES_REQUESTED** [review]
Thanks @ithake some very helpful updates - one small request also.

File: README.md (hunk 3 above)
Comment (ithake) [reply README.md]: Will do.  (That also answers a few questions we had!).

Comment (ithake) [reply README.md]: @ColinEberhardt  - given that the blog tool link to https://cz90l8ad7e.execute-api.eu-west-2.amazonaws.com/production/ still works - does that mean as a company we're being billed for keeping it running on AWS?

**Review by ColinEberhardt — APPROVED** [review]

**Comment by ColinEberhardt** [comment]
thanks @ithake

File: README.md (hunk 3 above)
Comment (ColinEberhardt) [reply README.md]: serverless ;-)

**Pull request merged**